### PR TITLE
PLAT-1213 Eagerly refresh invalidated or stubbed table rows

### PR DESCRIPTION
**Pull request by alexander-schmidt**

### CODE REVIEW

File: platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/AbstractDbTableRowFieldAccessor.java [hunk 1]
```diff
@@ -29,7 +30,7 @@ private void reloadRowIfNecessary() {
 
 		if (!row.table().getPrimaryKey().containsField(field) && !row.impermanent()) {
 			if (row.stub() || row.invalidated()) {
-				row.reload();
+				row.table().refreshAll(Collections.singleton(row));
```
Comment (daniel-klose) [inline platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/AbstractDbTableRowFieldAccessor.java:33]: Hm, is this a common occurence that we refresh a single row?
If it is, maybe we should add a default implementation of `refreshAll` with a varArgs-Param

File: platform-db/src/main/java/com/softicar/platform/db/runtime/cache/DbTableRowCacheBurstSizeCalculator.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		return this;
+	}
+
+	public DbTableRowCacheBurstSizeCalculator setBurstSizeCooldown(Duration burstSizeCooldown) {
+
+		this.burstSizeCooldown = burstSizeCooldown;
+		return this;
+	}
+
+	public int getNextBurstSize() {
```
Comment (oliver-richers) [inline platform-db/src/main/java/com/softicar/platform/db/runtime/cache/DbTableRowCacheBurstSizeCalculator.java:56]: i think that the name `get` rather implies that the method has no side-effect, so I suggest to rename it
```suggestion
	public int calculateNextBurstSize() {
```

**Review by oliver-richers — APPROVED** [review]

File: platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/AbstractDbTableRowFieldAccessor.java (hunk 1 above)
Comment (alexander-schmidt) [reply platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/AbstractDbTableRowFieldAccessor.java:33]: As this is rather uncommon (beyond unit tests), I don't really see a point in adding yet another method to `IDbBasicTable`.

File: platform-db/src/main/java/com/softicar/platform/db/runtime/cache/DbTableRowCacheBurstSizeCalculator.java (hunk 2 above)
Comment (alexander-schmidt) [reply platform-db/src/main/java/com/softicar/platform/db/runtime/cache/DbTableRowCacheBurstSizeCalculator.java]: ok

**Pull request merged**